wp-admin-ui/Admin.class.php

```diff
@@ -2058,6 +2058,18 @@ function get_data( $full = false ) {
 			$wpdb->show_errors( true );
 		}
 
+		// Check for the existence of multiple queries to run separated by ";;".
+		$multi_sql = preg_split( '/;{2,}\s+/', $sql, -1, PREG_SPLIT_NO_EMPTY );
+
+		// Check if we have multiple queries to run.
+		if ( 1 < count( $multi_sql ) ) {
+			// Get the last query and remove it from the array.
+			$sql = array_pop( $multi_sql );
+
+			// Execute the other queries altogether.
+			$multi = mysqli_multi_query( $wpdb->dbh, implode( '; ', $multi_sql ) );
```

Comment:
What's the intent of `$multi` here? Will it get used?

Reply:
It's a boolean response, so potentially could evaluate it somehow if it was false? Not sure if it's easy to integrate to your existing error handling.

Reply:
```suggestion
			// Check if we still have multiple queries to run.
			if ( 1 < count( $multi_sql ) ) {
				// Split the queries back up with a regular ";" delimiter.
				$multi_sql = implode( '; ', $multi_sql );

				// Run all queries.
				mysqli_multi_query( $wpdb->dbh, $multi_sql );
			} else {
				// Just get the query.
				$multi_sql = reset( $multi_sql );

				// Run the only query we have.
				$wpdb->query( $multi_sql );
			}
```